spec/60-trace-id-format.md

```diff
@@ -22,6 +22,9 @@ concerns](#privacy-considerations) of exposing unwanted information. Randomness
 also allows tracing vendors to base sampling decisions on `trace-id` field value
 and avoid propagating an additional sampling context.
 
+If the `random-trace-id` flag is set, the least significant 7 bytes of the
```

Comment:
The terminology "least significant" does not apply to a byte sequence. "right-most" is fine.